Stack/Opc.Ua.Core/Stack/State/ConditionState.cs

```diff
@@ -511,10 +748,24 @@ protected virtual void UpdateStateAfterDisable(ISystemContext context)
 
             UpdateEffectiveState(context);
         }
+
+        /// <summary>
+        /// Determines if this event is a branch
+        /// </summary>
+        /// <returns>true if branch</returns>
+        protected bool IsBranch()
+        {
+            return !(this.BranchId.Value.IsNullNodeId);
+        }
         #endregion
-        
+
         #region Private Fields
         private bool m_autoReportStateChanges;
+        /// <summary>
+        /// 
```

Comment:
missing comment

Reply:
I'm not concerned about the timeline.

As to an example, perhaps we can look at something at the next meeting.